cirq/sim/simulator_test.py

```diff
@@ -135,170 +123,26 @@ def steps(*args, **kwargs):
                                        qubit_order=qubit_order,
                                        initial_state=2)
     expected_results = [
-        cirq.SimulationTrialResult(
+        cirq.sim.simulator.SimulationTrialResult(
```

Comment:
Should just be `cirq.SimulationTrialResult`.

Reply:
done and cleaned up some imports